### PR TITLE
Dev/milestone2 docs

### PR DESCRIPTION
**Pull request by lav-mudgal-guavus**

### CODE REVIEW

File: core-plugins/docs/FileDelete-action.md [hunk 1]
```diff
@@ -1,30 +1,22 @@
-# File Delete Action
-
+# File Delete
 
 Description
 -----------
-Deletes a file or files.
+File Delete is a Guavus Enterprise Accelerator that is used to delete a file or files.
```
Comment (lav-mudgal-guavus) [inline core-plugins/docs/FileDelete-action.md]: This is an open source plugin.

File: core-plugins/docs/FileMove-action.md [hunk 2]
```diff
@@ -1,32 +1,25 @@
-# File Move Action
+# File Move
 
 
 Description
 -----------
-Moves a file or files.
+File Move is a Guavus Enterprise Accelerator that is used to moves a file or files.
```
Comment (lav-mudgal-guavus) [inline core-plugins/docs/FileMove-action.md]: This is an open source plugin.

File: core-plugins/docs/FileMove-action.md [hunk 3]
```diff
@@ -1,32 +1,25 @@
...
 
-**fileRegex:** Wildcard regular expression to filter the files in the source directory that will be moved.FileRegex pattern 
-would be simple search regex pattern which is used in any programming language like -- test.*/.csv -- (for move csv files start with test).
+**fileRegex:** The wildcard regular expression to filter the files in the source directory that will be moved. The fileRegex pattern would be a simple search regex pattern which is used in any programming language like -- test.*/.csv -- (for move csv files start with test).
 
-**continueOnError:** Indicates if the pipeline should continue if the move process fails. If all files are not 
-successfully moved, the action will not return the files already moved to their original locations.
+**continueOnError:** To specify if the pipeline should continue if the move process fails. If all files are not successfully moved, the action will not return the files already moved to their original locations.
```
Comment (lav-mudgal-guavus) [inline core-plugins/docs/FileMove-action.md]: "revert" instead of "return" ?

File: core-plugins/docs/XMLReader-batchsource.md [hunk 4]
```diff
@@ -1,30 +1,29 @@
-# XML Reader Batch Source
+# XML Reader
 
 
 Description
 -----------
-The XML Reader plugin is a source plugin that allows users to read XML files stored on HDFS.
+XML Reader is a Guavus Enterprise Accelerator that allows users to read XML files stored on HDFS.
```
Comment (deepak-maheshwari-ext) [inline core-plugins/docs/XMLReader-batchsource.md]: this is not "Guavus Enterprise Accelerator" . it is an "open source accelerator" .

File: transform-plugins/docs/XMLMultiParser-transform.md [hunk 5]
```diff
@@ -1,17 +1,15 @@
-# XML Multi Parser Transform
+# XML Multi Parser
 
 Description
 -----------
-The XML Multi Parser Transform uses XPath to extract fields from an XML document. It will generate records from
-the children of the element specified by the XPath. If there is some error parsing the document or building the record,
-the problematic input record will be dropped.
+XML Multi Parser is a Guavus Enterprise Accelerator that uses XPath to extract fields from an XML document. It will generate records from the children of the element specified by the XPath.
```
Comment (lav-mudgal-guavus) [inline transform-plugins/docs/XMLMultiParser-transform.md]: this is an open source plugin.

File: database-plugins/docs/Database-batchsource.md [hunk 6]
```diff
@@ -3,46 +3,41 @@
 
 Description
 -----------
-Reads from a database using a configurable SQL query.
-Outputs one record for each row returned by the query.
+Database is a Guavus Enterprise Accelerator that is used to read from a database using a configurable SQL query. It outputs one record for each row returned by the query.
```
Comment (deepak-maheshwari-ext) [inline database-plugins/docs/Database-batchsource.md]: this is not "Guavus Enterprise Accelerator" . it is an "open source accelerator" .

File: transform-plugins/docs/XMLMultiParser-transform.md [hunk 7]
```diff
@@ -25,21 +23,18 @@ Properties
 children of the elements referenced by the XPath. For example: /rss/channel/item.
 
 **schema:** The schema of records to output. Each field in the schema must be a child of the XML element referenced by
-the XPath. Currently only simply types are supported.
+the XPath. Currently, only simple types are supported.
 
 Conditions
```
Comment (lav-mudgal-guavus) [inline transform-plugins/docs/XMLMultiParser-transform.md]: we should change this to Note or something similar

File: database-plugins/docs/Database-batchsource.md [hunk 8]
```diff
@@ -3,46 +3,41 @@
...
 
 
 Use Case
 --------
-The source is used whenever you need to read from a database. For example, you may want
-to create daily snapshots of a database table by using this source and writing to
-a TimePartitionedFileSet.
+Consider a scenario wherein you want to create daily snapshots of a database table by using this source and writing to a TimePartitionedFileSet. This can be achieved by configuring the accelerator as described in the following section.
```
Comment (deepak-maheshwari-ext) [inline database-plugins/docs/Database-batchsource.md]: "create daily snapshots of a database table by" -> "read data from database"

File: transform-plugins/docs/JSONFormatter-transform.md [hunk 9]
```diff
@@ -3,19 +3,17 @@
 
 Description
 -----------
-Formats a Structured Record as JSON Object. Plugin will convert the Structured Record to a
-JSON object and write to the output record. The output record schema is a single field,
-either type ``STRING`` or type ``BYTE`` array.
+JSON Formatter is a Guavus Enterprise Accelerator that formats a Structured Record as JSON Object. This accelerator is used to convert the Structured Record to a JSON object and write to the output record. The output record schema is a single field,
```
Comment (neelesh-nirmal) [inline transform-plugins/docs/JSONFormatter-transform.md]: This is an open source plugin

File: transform-plugins/docs/CSVFormatter-transform.md [hunk 10]
```diff
@@ -3,23 +3,22 @@
 
 Description
 -----------
-Formats a Structured Record as a CSV Record. Supported CSV Record formats are ``DELIMITED``,
-``EXCEL``, ``MYSQL``, ``RFC4180``, and ``TDF``. When the format is ``DELIMITED``, one can specify different
-delimiters that a CSV Record should use for separating fields.
+CSV Formatter is a Guavus Enterprise Accelerator that formats a Structured Record as a CSV Record.
```
Comment (neelesh-nirmal) [inline transform-plugins/docs/CSVFormatter-transform.md]: This is an open source plugin.

File: transform-plugins/docs/Encryptor-transform.md [hunk 11]
```diff
@@ -1,36 +1,34 @@
-# Encryptor Transform
+# Field Encryptor
```
Comment (neelesh-nirmal) [inline transform-plugins/docs/Encryptor-transform.md:1]: 👍

File: transform-plugins/docs/Encryptor-transform.md [hunk 12]
```diff
@@ -1,36 +1,34 @@
...
 
 
 ## Description
-Encrypts one or more fields in input records using a java keystore 
-that must be present on all nodes of the cluster.
+Field Encryptor is a Guavus Enterprise Accelerator that is used to encrypt the fields present in the input records using a java keystore, which must be present on all nodes of the cluster.
```
Comment (neelesh-nirmal) [inline transform-plugins/docs/Encryptor-transform.md]: This is an open source plugin.

File: transform-plugins/docs/JSONParser-transform.md [hunk 13]
```diff
@@ -2,36 +2,31 @@
 
 ## Description
 
-Parses an input JSON event into a record. The input JSON event could be either a map of
-string fields to values or it could be a complex nested JSON structure. The plugin allows you
-to express JSON paths for extracting fields from complex nested input JSON.
+JSON Parser is a Guavus Enterprise Accelerator that parses an input JSON event into a record. The input JSON event can either map string fields to values or it can be a complex nested JSON structure. The accelerator allows you to mention JSON paths for extracting fields from complex nested input JSON.
```
Comment (deepak-maheshwari-ext) [inline transform-plugins/docs/JSONParser-transform.md]: it is an "ope source accelerator"

File: transform-plugins/docs/JSONParser-transform.md [hunk 14]
```diff
@@ -153,9 +142,7 @@ These operators are supported:
 
 #### Supported Functions
 
-The functions perform aggregations at the tail end of the path. The functions take the
-output of the expression as input for performing aggregations. The aggregation function
-can be only be applied where the expression results in an array. 
+The functions perform aggregations at the tail end of the path. The functions take the output of the expression as input for performing aggregations. The aggregation function \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\can be only be applied where the expression results in an array.
```
Comment (deepak-maheshwari-ext) [inline transform-plugins/docs/JSONParser-transform.md]: please remove these \'s

File: core-plugins/docs/FileMove-action.md (hunk 3 above)
Comment (rohit-raina-guavus) [reply core-plugins/docs/FileMove-action.md]: "restore"

File: transform-plugins/docs/Normalize-transform.md [hunk 15]
```diff
@@ -1,31 +1,25 @@
...
+# Normalize 
 
 
 Description
 -----------
-Normalize is a transform plugin that breaks one source row into multiple target rows.
-Attributes stored in the columns of a table or a file may need to be broken into multiple
-records: for example, one record per column attribute. In general, the plugin allows the
-conversion of columns to rows.
+Normalize is a Guavus Enterprise Accelerator that is used to break one source row into multiple target rows.
```
Comment (deepak-maheshwari-ext) [inline transform-plugins/docs/Normalize-transform.md]: it is an "open source Accelerator".

**Review by deepak-maheshwari-ext — CHANGES_REQUESTED** [review]

File: transform-plugins/docs/XMLToJSON-transform.md [hunk 16]
```diff
@@ -1,27 +1,22 @@
-# XML String to JSON String Converter Transform
+# XML to JSON
```
Comment (lav-mudgal-guavus) [inline transform-plugins/docs/XMLToJSON-transform.md]: # XML to JSON Converter

File: transform-plugins/docs/XMLToJSON-transform.md [hunk 17]
```diff
@@ -1,27 +1,22 @@
...
 
 
 Description
 -----------
-Accepts a field that contains a properly-formatted XML string and 
-outputs a properly-formatted JSON string version of the data.
+XML to JSON accepts a field containing a properly-formatted XML string and converts it into a properly-formatted JSON string.
```
Comment (lav-mudgal-guavus) [inline transform-plugins/docs/XMLToJSON-transform.md]: "XML to JSON converter" accelerator accepts a field..

File: transform-plugins/docs/XMLToJSON-transform.md [hunk 18]
```diff
@@ -1,27 +1,22 @@
...
 
 Use Case
 --------
-This plugin can be used with the Javascript transform for the parsing of 
-complex XML documents into parts. Once the XML is a JSON string, you 
-can convert it into a Javascript object using:
+XML to JSON can be used with the Javascript transform for the parsing of complex XML documents into parts. Once the XML is a JSON string, you can convert it into a Javascript object using:
```
Comment (lav-mudgal-guavus) [inline transform-plugins/docs/XMLToJSON-transform.md]: This accelerator can be used with

**Review by neelesh-nirmal — APPROVED** [review]
LGTM

**Review by deepak-maheshwari-ext — APPROVED** [review]
LGTM

**Pull request merged**